repair/repair.cc

```diff
@@ -1036,9 +1036,10 @@ static future<> repair_range(repair_info& ri, const dht::token_range& range) {
       }
       return ::service::get_local_migration_manager().sync_schema(ri.db.local(), neighbors).then([&neighbors, &ri, range, id] {
         return do_for_each(ri.cfs.begin(), ri.cfs.end(), [&ri, &neighbors, range] (auto&& cf) {
+            auto idx = std::distance(ri.cfs.begin(), std::find(ri.cfs.begin(), ri.cfs.end(), cf));
```

Comment:
Why not make `_cfs` store both the name and the UUID instead of doing this?

Reply:
I do not want to mess it up with the old partition level repair for backport reasons. Another reason, at some point we might want to store the uuids only.